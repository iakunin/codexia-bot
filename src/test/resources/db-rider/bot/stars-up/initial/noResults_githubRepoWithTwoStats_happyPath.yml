github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: "150713223"
        full_name: "test-project1/test-repo1"

codexia_project:
    -
        id: 5
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 12
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

github_repo_source:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "12"

github_repo_stat:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"type": "GITHUB_API", "stars": 402}'
        created_at: "2020-02-11 10:06:16.123456"
    -
        id: 3
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"type": "GITHUB_API", "stars": 459}'
        created_at: "2020-02-16 18:46:09.652645"

stars_up_result: []

codexia_review: []
