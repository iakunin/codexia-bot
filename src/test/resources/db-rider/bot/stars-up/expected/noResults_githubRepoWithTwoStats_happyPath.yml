stars_up_result:
    -
        github_repo_id: 1
        github_repo_stat_id: 3

codexia_review:
    -
        codexia_project_id: 5
        author: "STARS_UP"
        reason: "459"
        text: >-
            The repo gained 57 stars (from 402 to 459) in 5 days.
            See the stars history
            [here](https://star-history.t9t.io/#test-project1/test-repo1).
